analyzers/tests/SonarAnalyzer.UnitTest/Rules/TestMethodShouldContainAssertionTest.cs

```diff
@@ -163,6 +169,29 @@ public void ShouldNotThrowTest()
                 new TestMethodShouldContainAssertion(),
                 AdditionalTestReferences(NuGetMetadataReference.NUnit(testFwkVersion), fluentVersion));
 
+        [TestMethod]
+        public void TestMethodShouldContainAssertion_NUnit_NFluentLegacy() =>
+           Verifier.VerifyCSharpAnalyzer(@"
+using System;
+using NFluent;
+using NUnit.Framework;
+
+[TestFixture]
+public class Foo
+{
+    [Test]
+    public void Test1()
+    {
+        var x = 42;
```

Comment:
you could even remove lines 185-187 and 189 and just keep the minimum necessary code for the test

(applies to all)